src/promise-katas-2.js

```diff
@@ -47,7 +67,15 @@ const dog = () => {};
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all
 //
 
-const joke = () => {};
+const joke = () => {
+  return Promise.all([fetch('jokes', 'question'), fetch('jokes', 'answer')])
+  .then((res) => {
+    return{
+    question: res[0].joke, 
+    answer: res[1].answer
+    }
+  })
+};
```

Comment:
Same indentation as before, worth noting that you should have a gap after the return keyword and before the opening curly bracket:
```
const joke = () => {
  return Promise.all([
    fetch("jokes", "question"),
    fetch("jokes", "answer"),
  ]).then((res) => {
    return {
      question: res[0].joke,
      answer: res[1].answer,
    };
  });
};
```